godtools/App/Features/Onboarding/OnboardingQuickStartView.swift

```diff
@@ -0,0 +1,149 @@
+        
+        quickStartTableView.delegate = self
+        quickStartTableView.dataSource = self
+    }
+    
+    private func setupLayout() {
+        
+        titleLabel.text = viewModel.title
+        
+        endTutorialButton.setTitle(viewModel.endTutorialButtonTitle, for: .normal)
+        
+        setSkipButton()
+        
+        quickStartTableView.register(
+            UINib(nibName: OnboardingQuickStartCell.nibName, bundle: nil),
+            forCellReuseIdentifier: OnboardingQuickStartCell.reuseIdentifier
+        )
+        
+        quickStartTableView.separatorStyle = .none
+        quickStartTableView.rowHeight = UITableView.automaticDimension
+    }
+    
+    private func setupBinding() {
+        
+    }
+    
+    private func reloadData() {
+        
+    }
+    
+    private func setSkipButton() {
+        
+        let skipButtonPosition: ButtonItemPosition = .right
+        
+        if skipButton == nil {
+            skipButton = addBarButtonItem(
+                to: skipButtonPosition,
+                title: viewModel.skipButtonTitle,
+                style: .plain,
+                color: UIColor(red: 0.231, green: 0.643, blue: 0.859, alpha: 1),
+                target: self,
+                action: #selector(handleSkip(barButtonItem:))
+            )
+        }
+        else if let skipButton = skipButton {
+            addBarButtonItem(item: skipButton, barPosition: skipButtonPosition)
+        }
+    }
+    
+    @objc func handleSkip(barButtonItem: UIBarButtonItem) {
+        
+        viewModel.skipButtonTapped()
+    }
+    
+    @objc func handleEndTutorial() {
+        
+        viewModel.endTutorialButtonTapped()
+    }
+}
+
+// MARK: - UITableViewDelegate, UITableViewDataSource
+
+extension OnboardingQuickStartView: UITableViewDelegate, UITableViewDataSource {
+    
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return viewModel.quickStartItemCount
+    }
+    
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        
+        viewModel.quickStartCellTapped(index: indexPath.row)
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        
+        let cell: OnboardingQuickStartCell = quickStartTableView.dequeueReusableCell(
+            withIdentifier: OnboardingQuickStartCell.reuseIdentifier,
+            for: indexPath) as! OnboardingQuickStartCell
+                
+        let quickStartItem = viewModel.quickStartCellWillAppear(index: indexPath.row)
+        cell.configure(item: quickStartItem)
+        
+        return cell
+    }
+    
+    func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        
+        let headerView = UIView()
```

Comment:
Hey @reldredge71 is this doing anything?  I see in Figma this table list doesn't have any type of header UI so I am curious the intent of this.

Reply:
Ah, that was how I was trying to space out the cells before.  One StackOverflow thread said the only way they were able to add space was to put each cell in its own section and use the section headers for spacing.  I did what you suggested instead, to add the spacing in the .xib for the cell, but I forgot to remove this headerView.